docs/blog/author.yaml

```
@@ -285,3 +285,6 @@
 - id: Cory Mortimer
   bio: Software Engineer at Housecall Pro. Previously KnuEdge, Intuit. Passionate about front-end technologies, user-friendly design.
   avatar: avatars/cory-mortimer.png
+- id: Aileen Nowak
+  bio: JavaScript lover. JAMStack enthusiast. Junior Developer at Ghost.org
```

Why put junior, medior or senior in your bio? Just put Developer. Titles don't make sense 😉  I'm pretty sure you can call yourself medior or senior in some areas.

```suggestion
  bio: JavaScript lover. JAMStack enthusiast. Developer at Ghost.org
```

---

Haha! You're right! 😊